### DISCUSSION TITLE
Release of 2.0.0

### DISCUSSION THREAD

**Discussion by RafaelGB** [RafaelGB/obsidian-db-folder#218] [Announcements]
You can check the state of public 2.0.0 here https://github.com/users/RafaelGB/projects/4

Will be publish as soon as the Kanban is clear

**Comment by maxdomeuhd** [comment]
Hey, just want to thank you for your amazing work on this plugin, it's really helpful :)

**Comment by RafaelGB** [comment]
It is a pleasure!
there is still a lot of work to do to give the plugin more value
